scripts/base/protocols/ntp/main.zeek

```diff
@@ -0,0 +1,209 @@
+module NTP;
```

Comment:
This file in general needs a little work on formatting. The Zeek repo uses Whitesmiths style for all code, including zeek scripts. There's a number of tabs/spaces issues as well (prefer tabs).

Reply:
ok, done